homeworks/serhii.ripchanskyi_Ripchanskiy/11-dependency-injection/vechicle/src/app/app.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component } from '@angular/core';
+import { finalize } from 'rxjs/operators';
+
+import { Vehicle } from './models/';
+import { VehicleService } from './services';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
+})
+export class AppComponent {
+  title = 'vehicle';
+  vehicles: Vehicle[] = [];
+  areVehiclesLoading: boolean = false;
+
+  constructor(private vehicleService: VehicleService) {
+    this.getVehicles();
+  }
+
+  private getVehicles(): void {
+    this.vehicleService.getVehicles()
+      .pipe(finalize(() => this.areVehiclesLoading = true))
```

Comment:
You mixed up the statuses. When request is ended, users expected that `vehicles are loaded`. So please, update the correct status :smile: